src/tests/row.rs

```diff
@@ -11,6 +11,85 @@ mk_tests_default! {
     use std::str;
     use rand::{distributions::Standard, Rng};
 
+    #[test]
+    fn execute_procedure() -> Result<(), FbError> {
+        let mut conn = cbuilder().connect()?;
+
+        let (engine_version,): (String,) = conn.query_first(
+            "SELECT rdb$get_context('SYSTEM', 'ENGINE_VERSION') from rdb$database;",
+            (),
+        )?.unwrap();
+        if engine_version.starts_with("2.") {
+            return Ok(());
+        }
+
+        let ddl_procedure = "create or alter procedure get_value()
+                                returns (val int not null)
+                                as
+                                begin
+                                    val = 150;
+                                    suspend;
+                                end;";
+        conn.execute(ddl_procedure, ())?;
+
+        // Using select
+        let (val,): (i32,) = conn.query_first("select p.val from get_value p", ())?
+            .unwrap();
+        assert_eq!(150, val);
+
+        // Using exec proc
+        let (val,): (i32,) = conn.execute_returnable("execute procedure get_value", ())?;
+        assert_eq!(150, val);
+
+        Ok(())
+    }
+
+    #[test]
+    fn execute_block() -> Result<(), FbError> {
```

Comment:
```suggestion
    #[test]
    #[allow(clippy::float_cmp)]
    fn execute_block() -> Result<(), FbError> {
```

Clippy is complaining